packages/web/src/components/nav/desktop/NavPopupMenu.module.css

```diff
@@ -26,21 +25,26 @@
   position: relative;
 }
 
-.icon > svg path {
+.icon>svg path {
```

Comment:
was this prettified? seems like spaces would be desired

Reply:
my autosave did it. thx